### PR TITLE
Revise Quickstart to be testable with mdrip

### PR DESCRIPTION
**Pull request by jameshochadel**
@jgustie This isn't ready to merge yet, but I wanted to get your feedback early, particularly on the way I'm using `expect`. Does this look alright to you?

### CODE REVIEW

**Comment by jgustie** [comment]
You don't need to use `expect`, `redskyctl suggest` doesn't need to run interactively, it will take [assignments as arguments](https://redskyops.dev/docs/redskyctl/#redskyctl-suggest), e.g. `redskyctl suggest simple -A batchSize=256 --assign workers=2`; you can even just do a `--default rand` and get random suggestions.

**Comment by jameshochadel** [comment]
Nice!! I will do that.

File: docs/quickstart.md [hunk 1]
```diff
@@ -20,32 +20,46 @@ For more details, see [the installation guide](install.md).
...
+> ```
+
+You will be prompted to enter a value for each parameter in the experiment and a new trial will be created.
+
+You can monitor the progress using `kubectl`:
 
+<!-- @getTrials -->
 ```sh
-$ kubectl get trials
+kubectl get trials
```
Comment (robpacheco) [inline docs/quickstart.md]: What validates that this ran successfully?

Comment (jameshochadel) [reply docs/quickstart.md]: Good question.`mdrip` defines success as "every command exited with code 0". So this block will not validate anything other than that `kubectl get trials` runs without error. How about I change it to:

```
kubectl get trials | grep simple
```

...which will fail if nothing in the list matches?

Comment (jameshochadel) [reply docs/quickstart.md]: (There are alternative ways we could achieve that of course, but I think the `grep` technique strikes the right balance between making the test better and keeping the docs readable.)

Comment (jgustie) [reply docs/quickstart.md]: Why not `kubectl get trial simple`?

Comment (jameshochadel) [reply docs/quickstart.md]: Because `kubectl get` looks for an exact match, so that'll fail due to the suffix on the trial.

Comment (jgustie) [reply docs/quickstart.md]: Good point, how about `redskyctl get trials simple` then?

**Pull request closed without merging**